src/components/Comment.tsx

```diff
@@ -0,0 +1,81 @@
+/** @jsxImportSource @emotion/react */
+
+import { css } from "@emotion/react";
+import { ChangeEvent, FormEvent, useState } from "react";
+
+interface IComment {
+    name: string;
+    summary: string;
+}
+
+function Comment() {
+    const [comment, setComment] = useState<IComment[]>([]);
+    const [messageState, setMessageState] = useState("댓글이 아직 없어요");
+    const [nameState, setNameState] = useState<string>("");
+    const [summaryState, setSummaryState] = useState<string>("");
+
+    function onSubmitForm(event: FormEvent<HTMLFormElement>) {
+        event.preventDefault();
+        setComment((prev) => [...prev, { name: nameState, summary: summaryState }])
+        setMessageState("댓글들")
+        setNameState("")
+        setSummaryState("");
+    }
+
+    function onNameInput(event: ChangeEvent<HTMLInputElement>) {
+        setNameState(event.target.value)
+    }
+
+    function onSummaryInput(event: ChangeEvent<HTMLInputElement>) {
+        setSummaryState(event.target.value);
+    }
+
+    return (
+        <div>
+            <span css={text}>{messageState}</span>
+            <form css={inputWrapper} onSubmit={onSubmitForm}>
+                <input placeholder="닉네임" css={eachInput} value={nameState} onChange={onNameInput} />
+                <input placeholder="내용" css={eachInput} value={summaryState} onChange={onSummaryInput} />
+                <button css={buttonWrapper}>확인</button>
+            </form>
+            <div css={commentSort}>
+                {comment.map((todo: IComment) => (
```

Comment:
```suggestion
                {comment.map((todo) => (
```

여기서는 type을 따로 지정해주지 않으셔도 타입 추론이 되는데, 따로 지정해주신 이유가 있으실까요??

Reply:
그리고 댓글들이 리스트 렌더링되는 것인데, `todo`라는 이름은 부적절한 거 같아요